app-server/app/src/test/scala/AuthUtils.scala

```diff
@@ -0,0 +1,44 @@
+package com.azavea.rf
+
+import spray.json._
+import de.choffmeister.auth.common.JsonWebToken
+import akka.http.scaladsl.model.headers.{Authorization, OAuth2BearerToken}
+
+import com.azavea.rf.utils.Config
+
+object AuthUtils extends Config{
+  def generateToken(
```

Comment:
Spacing -- two lines between top-level objects/traits/classes and one line inside objects/traits/classes.

Also this needs a docstring I think.